www/src/pages/ru/usage/env-variables.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Переменные среды
+description: Начало работы с create-t3-app
+layout: ../../../layouts/docs.astro
+lang: ru
+isMdx: true
+---
+
+import Callout from "../../../components/docs/callout.tsx";
+
+Create-T3-App использует собственный пакет [@t3-oss/env-nextjs](https://env.t3.gg) и [Zod](https://github.com/colinhacks/zod) для проверки переменных среды во время выполнения _и_ во время сборки, предоставляя дополнительную логику в файле `src/env.mjs`:
+
+## env.mjs
+
+_TLDR; Если вам нужно добавить новую переменную среды, вы должны ее добавить как в `.env`, так и обозначить для нее валидатор в `src/env.mjs`._
+
+Это файл разделен на две части: схема и объект для деструктуризации и логика для валидации. Логику валидации трогать не стоит.
+
+```ts:env.mjs
+import { createEnv } from "@t3-oss/env-nextjs";
+import { z } from "zod";
+
+export const env = createEnv({
+  server: {
+    NODE_ENV: z.enum(["development", "test", "production"]),
+  },
+  client: {
+    // NEXT_PUBLIC_CLIENTVAR: z.string(),
+  },
+  runtimeEnv: {
+    NODE_ENV: process.env.NODE_ENV,
+  },
+});
+```
+
+T3 Env использует `createEnv`, который отвечает за создание схемы и будет включать главную логику валидации для клиентских и серверых переменных среды.
+
+<Callout type="info">
+  Для получения дополнительной информации о том, как `createEnv` работает изнутри, посмотрите документацию [T3
+  Env](https://env.t3.gg/docs/introduction) (На английском)
+</Callout>
+
+## Использование переменных среды
+
+Когда вы хотите использовать переменные среды, вы можете импортировать их из `env.mjs` и использовать их как вы обычно использовали бы. Если вы попробуете импортировать этот файл на стороне клиента и получить серверную переменную, вы получите ошибку:
```

Comment:
```suggestion
Когда вы хотите использовать переменные среды, вы можете импортировать их из `env.mjs` и использовать их как вы обычно использовали бы. Если вы импортируете этот файл на стороне клиента и попробуете получить серверную переменную, вы получите ошибку выполнения:
```
I feel like "try" that belonds to "accessing" is lost on my way reading to this point. Let's bring it closer to the initial meaning.